instructor/common.py

```diff
@@ -723,6 +729,7 @@ def western_blot_band_size(request):
             'formset': formset,
             'types_selected': types_selected,
             'assignment_name': assignment.name,
+            'num_error': json.dumps(error),
```

Comment:
Actually you may want to create the error message here instead and send it to the javascript to be shown. It makes the code in `instructor.js` less tied to this particular view